tests/interfaces/test_autograd.py

```diff
@@ -839,11 +847,7 @@ def cost_fn(x):
         )
         assert np.allclose(res, expected, atol=tol, rtol=0)
 
-        # Check that no deprecation warnigns are emitted due to trainable
-        # parameters with the Autograd interface
-        assert len(recwarn) == 0

```

Comment:
This is no longer relevant (no such warnings are emitted) and it was interfering with another warning raised.